public/js/containers/App.js

```diff
@@ -1,25 +1,27 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Actions } from 'jumpstate';
 import Page from 'components/Page';
 import Filters from 'components/Filters/Filters';
 import Charts from 'components/Charts/Charts';
 
-class App extends React.Component {
+class App extends Component {
     componentDidMount() {
+        Actions.fetchCommissioningDesksRedux();
         Actions.filterDesk(this.props.filterVals);
     }
 
     render() {
-        const { filterVals, isUpdating, charts } = this.props;
+        const { filterVals, isUpdating, charts, commissioningDesks } = this.props;
         return (
             <Page>
                 <Filters
                     filterVals={filterVals}
                     isUpdating={isUpdating}
+                    desks={commissioningDesks.desksList}
                 />
                 <Charts
-                    charts={charts}
+                    composerVsInCopy={charts.composerVsInCopy}
```

Comment:
I think I did it because I didn't want to have to pass `charts.composerVsInCopy.data` to the areachart, but rather just `composerVsInCopy.data`, and also because passing the single chart props data separately from the `Charts` component makes it easier in my head to read which type of data we will be passing down to the existing and future charts. So I think it's easier to read which kind of data we are using , but you are right that now stuff needs to be changed in more places and could be harder to remember. I am happy to change it back to how it was, I don't have strong feelings on this one :)